File: creational/java/builder/src/LowPackCarBuilder.java
public class LowPackCarBuilder extends CarBuilder{
    @Override
    public void buildEngine() { this.car.setEngine("1.4TFSI"); }
    @Override
    public void buildHeadLight() {
        this.car.setHeadlight("halogen");
    }
    @Override
    public void buildInterior() {
        this.car.setInterior("low pack");
    }
    @Override
    public void buildSunroof() { this.car.setSunroof("no sunroof"); }
}

File: creational/python/builder/LowPackCarBuilder.py
from CarBuilder import CarBuilder

class LowPackCarbuilder(CarBuilder):
    def __init__(self):
        CarBuilder.__init__(self)

    def buildEngine(self) ->None:
        self._car.setEngine("1.4TFSI")

    def buildInterior(self) ->None:
        self._car.setInterior("low pack interior")

    def buildHeadLight(self) ->None:
        self._car.setHeadlight("halogen")

    def buildSunroof(self) -> None:
        self._car.setSunroof("no sunroof")